tests/cft/test_r2_support_bundle.py

```diff
@@ -756,6 +756,99 @@ def test_31256(self):
         self.LOGGER.info("Successfully validated Motr rotating log files are as per "
                          "frequency configured for all pods")
 
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.lc
+    @pytest.mark.support_bundle
+    @pytest.mark.tags("TEST-34548")
+    def test_34548(self):
+        """
+        Validate support bundle “size_limit” filter
+        """
+        self.LOGGER.info("STARTED: Test to validate support bundle 'size_limit' filter")
+
+        self.LOGGER.info("Step 1: Generate support bundle")
+
+        pod_list = self.node_obj.get_all_pods(pod_prefix=constants.POD_NAME_PREFIX)
+        machine_id = self.node_obj.get_machine_id_for_pod(pod_list[0])
+        output = self.node_obj.execute_cmd(cmd=comm.KUBECTL_GET_POD_CONTAINERS.format(pod_list[0]),
+                                           read_lines=True)
+        container_list = output[0].split()
+
+        self.LOGGER.info("Generating support bundle for pod: %s", pod_list[0])
```

Comment:
Maybe trying to understand, is it only applicable for data pod?
Or else we can take list of pods for both data as well as server and then randomly choose one pod?

Reply:
yes, we can do that. Thats a good suggestion.
I will take that as improvement and will update this test also while raising other PR(for remaining tests)

Reply:
will update in this PR only

Reply:
Instead of random i have added check for both data and server pod